src/main/java/hudson/scm/SubversionSCM.java

```diff
@@ -1663,9 +1664,8 @@ public void load() {
             save();
         }
 
-        /*package*/ StandardCredentials migrateCredentials(ModelObject context, String legacyRealm, Credential legacyCredential)
+        /*package*/ StandardCredentials migrateCredentials(CredentialsStore store, String legacyRealm, Credential legacyCredential)
```

Comment:
It may be a breaking change in the case of SVN plugin. There's a code within same package in other plugins AFAIK